packages/synapse-react-client/src/components/GenericCard/TableRowGenericCard.test.tsx

```diff
@@ -51,7 +48,7 @@ const renderComponent = (
   return render(
     <QueryWrapper initQueryRequest={request}>
       <QueryVisualizationWrapper>
-        <GenericCard {...props} />
+        <TableRowGenericCard {...props} />
```

Comment:
The new TableRowGenericCard is functionally identical to the old GenericCard, so these tests just needed to use the new component.